packages/pwa-buildpack/src/WebpackTools/PWADevServer.js

```diff
@@ -41,11 +42,14 @@ const PWADevServer = {
         debug('configure() invoked', config);
         PWADevServer.validateConfig('.configure(config)', config);
         const devServerConfig = {
+            public: process.env.PWA_STUDIO_PUBLIC_PATH || '',
```

Comment:
found this [gem](https://webpack.js.org/configuration/dev-server/#devserverpublic)

Reply:
Heyyy, good catch! I absolutely approve of this